src/js/components/Navigation/SettingsSectionFooter.jsx

```diff
@@ -106,7 +106,15 @@ const TermsAndPrivacyText = styled('span')`
   color: #999;
   font-size: .9em;
   font-weight: 400;
+  .u-cursor--pointer:hover {
```

Comment:
This works! Thank you. I can't approve this pull request because of the other changes required in the other code, but these lines are ready to approve.

Reply:
To clarify, you can rollback the changes in src/js/components/Widgets/CopyLinkModal.jsx (the correct changes were approved in another pull request) The changes that remain in CopyLinkModal in this pull request are the prior changes, before they were updated.